enterprise_access/apps/subsidy_request/models.py

```diff
@@ -112,6 +116,10 @@ def bulk_update(cls, subsidy_requests, field_names, batch_size=SUBSIDY_REQUEST_B
 
     class Meta:
         abstract = True
+        index_together = [
+            ['uuid', 'state'],
+            ['user', 'enterprise_customer_uuid', 'state', 'course_id']
```

Comment:
Added this compound to cover this query specifically and a number of other use cases in this file: https://github.com/edx/enterprise-access/blob/b9ecdfacceb4912b908ad74bfd1894f7169fa5fa/enterprise_access/apps/api/v1/views.py#L444